apps/cookbook/src/app/examples/modal-example/modal-example.component.ts

```diff
@@ -12,7 +13,7 @@ const config = {
 <button kirby-button (click)="showModalWithFooter()">Show modal with footer</button>`,
   footerTemplate: `<p>Some content of the embedded component</p>
 ...
-<kirby-modal-footer>
+<kirby-modal-footer [snapToKeyboard]="false">
```

Comment:
Does it make sense to include this in the snippet if it defaults to false anyway?
```suggestion
<kirby-modal-footer>
```